storage/migrations/29_fix_msig_transaction_params.go

```diff
@@ -0,0 +1,19 @@
+package migrations
+
+import (
+	"github.com/go-pg/migrations/v8"
+)
+
+// Schema version 29 fixes https://github.com/filecoin-project/sentinel-visor/issues/459
+
+func init() {
+	up := batch(`
+	ALTER TABLE multisig_transactions ALTER COLUMN params DROP NOT NULL;
+`)
+
+	down := batch(`
+	ALTER TABLE multisig_transactions ALTER COLUMN params SET NOT NULL;
```

Comment:
Since any table we reverse this migration on will now likely have null values and will cause the `down` to fail. You'll probably need to make this a no-op or find a reasonable non-null default to replace them with before running the `ALTER`.

Reply:
Strictly speaking we should keep it as you have written

Reply:
Many down migrations cannot work against populated tables, but we should include them for correctness. I struggle to think of good scenarios for down migrations and I'm now thinking that we should only ever increment the schema version. A "downgrade" would be a new migration that removes the thing we added in an earlier one.